test.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import unittest
+
+
+class TestServerFunctionality(unittest.TestCase):
+
+	def setUp(self):
+		self.url= 'http://ec2-52-11-248-121.us-west-2.compute.amazonaws.com/api/v1'
+		self.loginval= {"login":"test", "password":"test"}
+
+	def test_1Login(self):
+		requestLogin = requests.post(self.url+'/login', json= self.loginval)
+		Token=(requestLogin.json().get('data')).get('tokenKey')
+		self.assertEqual(requestLogin.json().get('status'), 1)
+		EmailVerif= (requestLogin.json().get('data')).get('emailVerified')
+		self.assertEqual(EmailVerif,1)
+		userid= (requestLogin.json().get('data')).get('user_id')
+		self.assertEqual(EmailVerif,1)
+		return Token
+
+	def test_2Items(self):
+		Token= self.test_1Login()
+		headers={'Token-Key':'%s'%Token }
+		requestItems = requests.get(self.url+'/items', headers=headers)
+		self.assertEqual(requestItems.json().get('status'), 1)
+		todoData= (requestItems.json().get('data')).get('todoData')
+		for x in range(0,len(todoData)):
```

Comment:
Test should be independent, if you get and check data then you need to set it for this test, or other test can corrupt work of this test